build/README.md

```diff
@@ -52,6 +52,8 @@ We recommend [Balena Etcher](https://www.balena.io/etcher/)
 
 Visit http://embassy.local from any web browser
 
+You will need to enter your product key, which is created at build in the file `product_key.txt`
+
 We recommend [Firefox](https://www.mozilla.org/firefox/browsers)
```

Comment:
This line is in reference to "Visit http://embassy.local from...", so should be appended to it.